collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/FormMetadataSettingsTest.kt

```diff
@@ -40,30 +40,67 @@ class FormMetadataSettingsTest {
         .around(rule)
 
     @Test
-    fun settingMetadata_letsThemBeIncludedInAForm() {
+    fun metadataShouldBeDisplayedInPreferences() {
```

Comment:
@grzesiek2010 it looks to me like this test fails if I add it on `master`. I'm guessing that's expected (as it looks like it verifies the original problem), but just wanted to check?

Reply:
Yes you are right.